ido-vertical-mode.el

```diff
@@ -87,7 +87,7 @@ so we can restore it when turning `ido-vertical-mode' off")
 (defvar ido-vertical-count-active nil
   "Used internally to track whether we're already showing the count")
 
-(defcustom ido-vertical-define-keys 'C-n-and-C-p-only
```

Comment:
Oh! And a NEWS file would avoid surprises for current users.  If I am not mistaken, a NEWS file can be generated by commit messages.